### ISSUE TITLE
Anyone running one vps?

### ISSUE THREAD

**Issue by Al1-mert** [AiYinZiLeGong/PUBG-Radar#134]
Is it possible to run on a vps?

**Comment by nengbowan** [comment]
no, my friends said : middle pc and game pc must be on the same vlan

**Comment by Al1-mert** [comment]
hamachi maybe? is there any vps with opengl support?

**Comment by bhairawa** [comment]
Amazon EC2 have OpenGL support, or you can run your own host on VMware ESXi. But this does not help anyway, since this radar does not works in RDP session (this issue i experience personally). Possible run with VNC but picture updates is too slow for normal usage.

**Comment by Al1-mert** [comment]
bhairawa, try to connect with spacedesk or team viewer maybe?

**Comment by bhairawa** [comment]
Just try to play via teamviewer, etc any video and you will see its capable or not.

**Comment by Al1-mert** [comment]
try spacedesk then, its really fast on same lan. i also tried broadcosting my radar, it running fine but there is 10 sec delay no matter what you do.
